modules/internal/OwnedObject.chpl

```diff
@@ -259,11 +240,9 @@ module OwnedObject {
        Deletes the previously managed object, if any.
      */
     pragma "leaves this nil"
+    @deprecated(notes="owned.clear is deprecated - please use :proc:`owned.release` instead")
     proc ref clear() {
```

Comment:
I would like the "instead" clause to read something like "please assign nil instead".